### PR TITLE
Custom Constraint Samplers

### PR DESCRIPTION
**Pull request by mlautman**
still a stub, but at least it is no longer poorly

### CODE REVIEW

File: doc/custom_constraint_samplers/custom_constraint_samplers_tutorial.rst [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
 
 Creating a constraint sampler
 -----------------------------
 
-* Create a ROBOT_moveit_plugins package and within that a sub-folder for your YOURROBOT_constraint_sampler plugin.
-  Modify the template provided by hrp2jsk_moveit_plugins/hrp2jsk_moveit_constraint_sampler_plugin
-* In your ROBOT_moveit_config/launch/move_group.launch file, within the <node name="move_group">, add
-  the parameter:
+* Create a ``ROBOT_moveit_plugins`` package and within that a sub-folder for your ``ROBOT_constraint_sampler`` plugin. Modify the template provided by ``hrp2jsk_moveit_plugins/hrp2jsk_moveit_constraint_sampler_plugin``
```
Comment (davetcoleman) [inline doc/custom_constraint_samplers/custom_constraint_samplers_tutorial.rst]: hrp2jsk --> ROBOT

Comment (mlautman) [reply doc/custom_constraint_samplers/custom_constraint_samplers_tutorial.rst]: Change made. Is there a link to examples or template files that I can add to the tutorial?

Comment (davetcoleman) [reply doc/custom_constraint_samplers/custom_constraint_samplers_tutorial.rst]: I just emailed the JSK lab in Tokyo to see if I can open source this effort I did for them a few years ago. At the time the URDF was closed source so we could not

Comment (mlautman) [reply doc/custom_constraint_samplers/custom_constraint_samplers_tutorial.rst]: thanks!

**Pull request merged**